librz/include/rz_arch.h

```diff
@@ -41,6 +49,13 @@ RZ_API bool rz_arch_profiles_init(RzArchTarget *c, const char *cpu, const char *
 RZ_API void rz_arch_profile_add_flag_every_io(RzArchProfile *profile, RzFlag *flags);
 RZ_API bool rz_arch_load_profile_sdb(RzArchTarget *t, const char *path);
 
+RZ_API RZ_OWN RzArchPlatformItem *rz_arch_platform_item_new(RZ_NULLABLE const char *name);
+RZ_API RZ_OWN RzArchPlatformTarget *rz_arch_platform_target_new();
+RZ_API void rz_arch_platform_target_free(RzArchPlatformTarget *target);
+RZ_API void rz_arch_platform_item_free(RzArchPlatformItem *item);
+RZ_API bool rz_arch_load_platform_sdb(RzArchPlatformTarget *t, const char *path);
+RZ_API bool rz_arch_platform_init(RzArchPlatformTarget *t, const char *arch, const char *cpu, const char *platform, const char *dir_prefix);
```

Comment:
Also please add these `RZ_NONNULL` here as well.

Reply:
Where exactly should I add `RZ_NONNULL` here?

Reply:
To `arch`, `cpu`, etc

Reply:
RZ_NONNULL goes to any value that you check when you use `rz_return[_val]_if()`